src/openvic-simulation/modifier/StaticModifierCache.hpp

```diff
@@ -1,62 +1,65 @@
 #pragma once
 
+#include "openvic-simulation/dataloader/NodeTools.hpp"
+#include "openvic-simulation/modifier/Modifier.hpp"
 #include "openvic-simulation/utility/Getters.hpp"
 
 namespace OpenVic {
-	struct Modifier;
 	struct ModifierManager;
 
 	struct StaticModifierCache {
 		friend struct ModifierManager;
 
 	private:
 		// Country modifiers
-		Modifier const* PROPERTY(very_easy_player);
-		Modifier const* PROPERTY(easy_player);
-		Modifier const* PROPERTY(hard_player);
-		Modifier const* PROPERTY(very_hard_player);
-		Modifier const* PROPERTY(very_easy_ai);
-		Modifier const* PROPERTY(easy_ai);
-		Modifier const* PROPERTY(hard_ai);
-		Modifier const* PROPERTY(very_hard_ai);
+		Modifier PROPERTY(very_easy_player);
+		Modifier PROPERTY(easy_player);
+		Modifier PROPERTY(hard_player);
+		Modifier PROPERTY(very_hard_player);
+		Modifier PROPERTY(very_easy_ai);
+		Modifier PROPERTY(easy_ai);
+		Modifier PROPERTY(hard_ai);
+		Modifier PROPERTY(very_hard_ai);
 
-		Modifier const* PROPERTY(base_modifier);
-		Modifier const* PROPERTY(war);
-		Modifier const* PROPERTY(peace);
-		Modifier const* PROPERTY(disarming);
-		Modifier const* PROPERTY(war_exhaustion);
-		Modifier const* PROPERTY(infamy);
-		Modifier const* PROPERTY(debt_default_to);
-		Modifier const* PROPERTY(bad_debter);
-		Modifier const* PROPERTY(great_power);
-		Modifier const* PROPERTY(secondary_power);
-		Modifier const* PROPERTY(civilised);
-		Modifier const* PROPERTY(uncivilised);
-		Modifier const* PROPERTY(literacy);
-		Modifier const* PROPERTY(plurality);
-		Modifier const* PROPERTY(generalised_debt_default);
-		Modifier const* PROPERTY(total_occupation);
-		Modifier const* PROPERTY(total_blockaded);
+		Modifier PROPERTY(base_modifier);
+		Modifier PROPERTY(war);
+		Modifier PROPERTY(peace);
+		Modifier PROPERTY(disarming);
+		Modifier PROPERTY(war_exhaustion);
+		Modifier PROPERTY(infamy);
+		Modifier PROPERTY(debt_default_to);
+		Modifier PROPERTY(great_power);
+		Modifier PROPERTY(secondary_power);
+		Modifier PROPERTY(civilised);
+		Modifier PROPERTY(uncivilised);
+		Modifier PROPERTY(literacy);
+		Modifier PROPERTY(plurality);
+		Modifier PROPERTY(total_occupation);
+		Modifier PROPERTY(total_blockaded);
+
+		 //country event modifiers
```

Comment:
```suggestion
		// Country event modifiers
```
Remove the space before the `//` and change the comment to match the format of `// Country modifiers` above and `// Province modifiers` below